docs/csharp/whats-new/configure-language-version.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Select the C# language version - C# Guide
+description: Configure the compiler to perform syntax validation using a specific compiler version
+ms.date: 05/24/2018
+---
+
+# Select the C# language version
+
+The C# compiler defaults to the latest major version of the language that has been released. You may choose to compile any project using a new point release of the language. Choosing a newer version of the language enables your project to make use of the latest language features. In other scenarios, you may need to validate that a project compiles cleanly when using an older version of the language.
+
+This capability decouples the decision to install new versions of the SDK and tools in your development environment from the decision to incorporate new language features in a project. You can install the latest SDK and tools on your build machine. Each project can be configured to use a specific version of the language for its build.
+
+There are several ways to set the language version:
+
+- Rely on a [Visual Studio quick action](#visual-studio-quick-action)
+- Set the language version in [Visual Studio UI](#set-the-language-version-in-visual-studio)
+- Manually edit your [**.csproj** file](#edit-the-csproj-file)
+- Set the language version [for multiople projects in a subdirectory](#configure-multiple-projects)
+- Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option)
+
+## Visual Studio quick action
+
+Visual Studio helps you determine the language version you need. If you use a language feature that is not available for the currently configured version, Visual Studio shows a potential fix to update the language version for the project.
+
+## Set the language version in Visual Studio
+
+You can set the version in Visual Studio. Right-click on the project node in Solution Explorer and select **Properties**. Select the **Build** tab and select the **Advanced** button. In the dropdown, select the version. The following image shows the "latest" setting.
+
+> [!NOTE]
+> If you use the Visual Studio IDE to update your csproj files, the IDE
+> creates separate nodes for each build configuration. You'll typically
+> set the value the same in all build configurations, but you need to
+> set it explicitly for each build configuration, or select "All Configurations"
+> when you modify this setting. You'll see the following in your csproj file:
+
+```xml
+<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+  <LangVersion>latest</LangVersion>
+</PropertyGroup>
+
+<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+  <LangVersion>latest</LangVersion>
+</PropertyGroup>
+```
+
+## Edit the csproj file
+
+You can set the language version in your **.csproj** file. Add the following element:
```

Comment:
instead of "the following", "an element with a value like the following" perhaps